lighthouse-core/test/audits/maskable-icon-test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const MaskableIconAudit = require('../../audits/maskable-icon.js');
+const assert = require('assert');
+const manifestParser = require('../../lib/manifest-parser.js');
+
+const manifestSrc = JSON.stringify(require('../fixtures/manifest.json'));
+const manifestWithMaskableSrc = JSON.stringify(require('../fixtures/manifest-maskable-icon.json'));
+const EXAMPLE_MANIFEST_URL = 'https://example.com/manifest.json';
+const EXAMPLE_DOC_URL = 'https://example.com/index.html';
+
+/**
+ * @param {string}
+ */
+function generateMockArtifacts(src = manifestSrc) {
+  const exampleManifest = manifestParser(src, EXAMPLE_MANIFEST_URL, EXAMPLE_DOC_URL);
+
+  return {
+    WebAppManifest: exampleManifest,
+    InstallabilityErrors: {errors: []},
+  };
+}
+
+function generateMockAuditContext() {
+  return {
+    computedCache: new Map(),
+  };
+}
+
+/* eslint-env jest */
+
+describe('Maskable Icon Audit', () => {
+  const context = generateMockAuditContext();
+
+  it('fails when the manifest fails to be parsed', async () => {
+    const artifacts = generateMockArtifacts();
+    artifacts.WebAppManifest = null;
+
+    const auditResult = await MaskableIconAudit.audit(artifacts, context);
+    assert.equal(auditResult.score, 0);
```

Comment:
🚨 totally optional personal appeal below 🚨

since this is a brand new test file I'll attempt to pitch you on why new tests should use [expect](https://jestjs.io/docs/en/expect) instead of assert :)

while it has pretty much no impact on this particular file, `expect` gives us some nice powers over assert, namely

- partial matching 
  `expect({foo: 1, ...giantListOfOtherProperties}).toMatchObject({foo: 1})` will pass and on failure still print out the rest of the object for better debugging
- better array length debugging
 `expect(arr).toHaveLength(5)` will print the contents of the array on failure for debugging instead of just saying `Expected 5 got 4`
- snapshot testing
  `expect(someObject).toMatchInlineSnapshot()` is super handy for tests where we don't necessarily have an absolute correct answer but we want to be aware when anything affects the behavior (think lantern metrics, complicated byte or time savings estimation logic, etc) jest automatically manages these golden file expectations  inline and can update them with the `-u` flag to jest
- mock assertions
  `expect(jest.fn()).toHaveBeenCalledWith('foo')`, much easier to read and grok than the manual mock functions we have crafted elsewhere with `assert`

on the whole, most of the codebase is still `assert` and when adding individual tests that don't benefit from `expect` to thsoe files, totally expected to follow the surrounding style, but in new places where we might want to reap the benefits of expect I strongly support the use `expect`

and that's my spiel, feel free to decide for yourself moving forward when to use what appropriately :)

Reply:
You've sold me! Made the switch.